client/ayon_harmony/plugins/publish/extract_render.py

```diff
@@ -116,7 +108,22 @@ def process(self, instance):
 
         self.log.debug(output.decode("utf-8", errors="backslashreplace"))
 
-        # Generate representations.
+        # Consider the collection with the most files as the render
+        # Others could be thumbnails
+        self.log.debug(f"collections: {collections}")
+        if len(collections) > 1:
+            for col in reversed(collections):
+                if len(list(col)) > 1:
```

Comment:
```suggestion
                if len(col.indexes) > 1:
```

Reply:
But is there really no better way to _know_ what files were actually rendered? :')

Reply:
I don't know, I copied this from the previous version of the code. I think we could either parse the process output, or build a list of expected files with the parameters passed to the `Popen` process. But this is an optimization not directly linked to this PR.